hazenlib/tasks/acr_geometric_accuracy.py

```diff
@@ -42,6 +45,65 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self.ACR_obj = ACRObject(self.dcm_list)
 
+    def detect_best_rotation(self, ref_img):
```

Comment:
Rather than adding `detect_best_rotation` as a method for every class I think it would best to move this to a method for the ACR object.

This way we only have one implementation and if we ever need to make changes to this method in the future we only need to change one method rather than one method for every class - I hope that makes sense.